src/game.py

```diff
@@ -26,22 +28,18 @@ def __init__(self, ui=UserInterface(), message=Message()):
         self.playing = True
         self.play_against_computer = False
 
-    def valid_computer_move(self):
-        computer_move = self.computer_player.computer_input()
-        valid_computer_move = self.validator.spot_is_available(
-            self.game_board, computer_move
-        )
-        while not valid_computer_move:
-            computer_move = self.computer_player.computer_input()
-            valid_computer_move = self.validator.spot_is_available(
-                self.game_board, computer_move
-            )
-        return computer_move
+    def valid_computer_move(self, move):
+        valid_computer_move = self.validator.spot_is_available(self.game_board, move)
+        if not valid_computer_move:
```

Comment:
that looks more like it 😄

Reply:
further feedback; probably should not a variable named the same name as the function it is inside of.

I suggest just in-lining that variable

Reply:
```python
    def valid_computer_move(self, move):
        if not self.validator.spot_is_available(self.game_board, move):
            return self.valid_computer_move(self.computer_player.computer_input())
        return move

Reply:
to further simplify the game class, you can do this refactoring on several other methods. (maybe in a campsite pr, because that change is out of scope for the ticket you are working on).

Reply:
Yeahhhhh nice!! 💃